src/main/java/model/figure/Circle.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Circle extends Figure {
```

Comment:
The class name 'Circle' is appropriate as it is a noun, which is good practice for naming classes.